packages/mds-db/schema.ts

```diff
@@ -180,6 +209,59 @@ const TABLE_COLUMNS: { [T in TABLE_NAME]: Readonly<COLUMN_NAME[]> } = {
   [TABLE.migrations]: [COLUMN.id, COLUMN.migration, COLUMN.timestamp],
   [TABLE.policies]: [COLUMN.id, COLUMN.policy_id, COLUMN.policy_json],
   [TABLE.policy_metadata]: [COLUMN.id, COLUMN.policy_id, COLUMN.policy_metadata],
+  [TABLE.reports_device_states]: [
+    COLUMN.id,
+    COLUMN.vehicle_type,
+    COLUMN.type,
+    COLUMN.timestamp,
+    COLUMN.device_id,
+    COLUMN.provider_id,
+    COLUMN.recorded,
+    COLUMN.annotation_version,
+    COLUMN.annotation,
+    COLUMN.gps,
+    COLUMN.service_area_id,
+    COLUMN.charge,
+    COLUMN.state,
+    COLUMN.event_type,
+    COLUMN.event_type_reason,
+    COLUMN.trip_id
+  ],
```

Comment:
Seems like these new tables ought to have a `recorded` column for consistency and tracking purposes.

Reply:
Hmm is there a better way to organize these? I don't really want that prefix on all our columns, especially if we are planning on deprecating the old event/telem tables down the line

Reply:
I am suggesting `COLUMN.recorded` should be included in each of the three reports tables as a record of when rows are written. It looks like you have it on 1/3 presently.

Reply:
Oh I see. Since the other two tables are populated on a cron schedule I'm not sure of the benefit. The reports_providers table has a start_time column that would be somewhat analogous.

Reply:
Consistency is a benefit.

Reply:
Ok, will add.